homeassistant/components/scrape/sensor.py

```diff
@@ -144,6 +147,45 @@ async def async_setup_platform(
     async_add_entities(entities)
 
 
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Set up the Scrape sensor entry."""
+    entities: list = []
+
+    coordinator: ScrapeCoordinator = hass.data[DOMAIN][entry.entry_id]
+    config = dict(entry.options)
+    for sensor in config["sensors"]:
+        sensor_config: ConfigType = vol.Schema(
+            TEMPLATE_SENSOR_BASE_SCHEMA.schema, extra=vol.ALLOW_EXTRA
+        )(sensor)
+
+        name: str = sensor_config[CONF_NAME]
+        select: str | None = sensor_config.get(CONF_SELECT)
+        attr: str | None = sensor_config.get(CONF_ATTRIBUTE)
+        index: int = int(sensor_config[CONF_INDEX])
+        value_string: str | None = sensor_config.get(CONF_VALUE_TEMPLATE)
+
+        value_template: Template | None = (
+            Template(value_string, hass) if value_string is not None else None
+        )
```

Comment:
```suggestion
        value_template: Template | None = sensor_config.get(CONF_VALUE_TEMPLATE)
        if value_template is not None:
            value_template.hass = hass
```

Reply:
It's not a template. It's a string so we would need to make it a template

Reply:
Ah yes!